control/patch.yaml

```diff
@@ -0,0 +1,41 @@
+openapi: 3.1.0
+
+info:
+  title: Control patch models
+  description: >-
+    All control patch schemas
+  version: ^0.0.0
+
+components:
+  schemas:
+    Control.Patch:
```

Comment:
Unlike any other control API, this action changes the configuration (which will also reflect in get_config). Need to reflect that in the description, to avoid any ambiguity. 
On second thought, if the API should be moved under the `configuration` group - and renamed to `update_config` - since the other two APIs in the group are also related to configuration and it will be obvious that the configuration would be modified and reflected in get_config?